src/util.ts

```diff
@@ -122,3 +122,8 @@ export const getStartOfMonthsFromNow = (
   const month = date.getUTCMonth()
   return new Date(Date.UTC(year, month + months, 1, 0, 0, 0, 0))
 }
+
+export const safeParseFloat = (val: string): number => {
+  if (val === '') return 0
```

Comment:
I was expecting this:

```ts
const parsed = parseFloat(val)
if (isNaN(parsed)) return 0
return parsed
```

That would catch everything, and not just blank strings. If the intent is to just filter blank strings. then maybe we should document that instead:


```ts
/**
 * Parses number strings, treating blank inputs as 0
 */
```

Otherwise, the word "safe" seems to imply that it always returns valid values, regardless of input.

Reply:
I initially did the isNaN check but switched to only check blank strings since that "feels" like it could be interpreted as 0. If parseFloat actually got passed a "NaN" or `null` which would also return `null` then I was expecting to return null since that is not easily considered a 0 value. This matches biggystring which currently interprets empty string as a zero but not "NaN" or null.

Reply:
Then add the comment, and I will approve it.

Reply:
Fine, I'll do it myself.